src/inference/hmc.jl

```diff
@@ -663,9 +607,9 @@ observe(spl::Sampler{<:Hamiltonian},
 #### Default HMC stepsize and mass matrix adaptor
 ####
 
-function AHMCAdaptor(alg::AdaptiveHamiltonian; init_ϵ=alg.init_ϵ)
```

Comment:
This seems to be another overwritten of https://github.com/TuringLang/Turing.jl/pull/849

Reply:
Good catch, thanks.